### PR TITLE
Higher quality code.

### PR DESCRIPTION
**Pull request by abdelrahman0w**
I tried to refactor the code in some file lines to make higher quality.

### CODE REVIEW

File: lisa/features/serial_console.py [hunk 1]
```diff
@@ -113,6 +102,20 @@ def get_console_log(
 
         return self._cached_console_log.decode("utf-8", errors="ignore")
 
+    def _extracted_from_get_console_log_9(self):
```
Comment (squirrelsc) [inline lisa/features/serial_console.py:105]: What's the reason to export a method? The name isn't meaningful.

Comment (glima) [reply lisa/features/serial_console.py:105]: Hero commit, really appreciated. Is this coming from an automated tool, however? What's up with this weird name? :)

File: lisa/node.py [hunk 2]
```diff
@@ -477,15 +470,9 @@ def from_existing(
         environment_name: str,
         base_log_path: Optional[Path] = None,
     ) -> Node:
-        node = Node.create(
-            index=len(self._list),
-            runbook=node_runbook,
-            logger_name=environment_name,
-            base_log_path=base_log_path,
+        return self._extracted_from_from_requirement_7(
```
Comment (glima) [inline lisa/node.py:473]: Yep, pretty sure this is not human-generated. Please fix the random naming, then.

Comment (glima) [reply lisa/node.py:473]: Is this https://github.com/python-rope/rope? Something else?

Comment (abdelrahman0w) [reply lisa/node.py:473]: @glima 
Hi,
No, I really didn't use that tool. However, I use an extension that suggest modifications, I think I shouldn't have used it. I am really sorry. But I really did changes by myself, the extension just suggested minor changes. I am sorry.
For the name as you mentioned, I just didn't know what to write, and I don't know if there is guidelines to follow in naming branches, but I am sorry if the name is not appropriate.

Comment (squirrelsc) [reply lisa/node.py:473]: @Abdelrahman0W  please remove all function extraction changes, I will have a look on other changes.

Comment (abdelrahman0w) [reply lisa/node.py:473]: @squirrelsc 
Okay, I will. Thank you so much

**Comment by squirrelsc** [comment]
@Abdelrahman0W I have to close this PR, because there is no response long time, and there are a lot of conflicts. Feel free to solve the comments and reopen it.

**Pull request closed without merging**